xcode/Subconscious/Shared/Components/Editor/EditorModalSheetDetailView.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  MemoEditorDetailView.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 9/20/21.
+//
+
+import SwiftUI
+import os
+import ObservableStore
+import Combine
+
+// MARK: View
+struct EditorModalSheetDetailView: View {
```

Comment:
I forked the existing EditorDetailView for the modal sheet, it's powered by the same underlying model.

When we're ready to cut over we can flatten this out and remove the unused features.